code/stat159lambda/simulations/correlation_simulation.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+from scipy import stats
+import matplotlib.pyplot as plt
+from stat159lambda.config import REPO_HOME_PATH
+
+
+def simulate_inter_run_correlation():
+    subj_1 = np.concatenate((np.random.normal(5, 2, 100), np.random.normal(
+        10, 2, 100)))
```

Comment:
Yes -- the purpose of this simulation is to provide context in our paper to motivate handling how arbitrary runs can affect correlation